package.json

```diff
@@ -76,7 +76,8 @@
     "redux-persist": "^4.3.1",
     "redux-persist-transform-encrypt": "^1.0.2",
     "redux-thunk": "^2.2.0",
-    "reselect": "^3.0.1"
+    "reselect": "^3.0.1",
+    "redux-devtools-extension": "^2.13.2"
```

Comment:
Can we have this as a `devDependency`?

Reply:
Hmm, actually I looked at other repositories, and there the dependency `redux-devtools-extension` is in just `dependencies`. Given that this only works when the debugging mode is enabled, then I understand that there is no point in adding a script command, I may be mistaken, but as I understand it, this is so.

Reply:
Perfect no worries, if you don't think it makes sense to add a script then that's perfectly fine :)

Yeah and I've noticed the same with `redux-logger` and I'm still no sure why those two usually live as dependencies :thinking: Let's circle back to that after some time.